rdbi-recipes/src/main/java/com/lithium/dbi/rdbi/ratelimiter/StrictRateLimiter.java

```diff
@@ -0,0 +1,102 @@
+package com.lithium.dbi.rdbi.ratelimiter;
+
+import com.google.common.base.Joiner;
+import com.lithium.dbi.rdbi.Handle;
+import com.lithium.dbi.rdbi.RDBI;
+import org.apache.commons.codec.digest.DigestUtils;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.exceptions.JedisDataException;
+
+import java.util.OptionalLong;
+
+/**
+ * Rate limiter implementation that will never allow
+ * greater than the configured requests/second
+ */
+public class StrictRateLimiter implements RateLimiter {
```

Comment:
I'm assuming that you're just renaming this impl class from ```RateLimiter``` to ```StrictRateLimiter``` and not changing any of the impl details, right?